### DISCUSSION TITLE
Issues making tailwindcss plugin gen css files for website

### DISCUSSION THREAD

**Discussion by illyrius666** [lumeland/lume#670] [Q&A] [answered]
using the plugin i dont get my website styled, while using the cdn version does. what am i doing wrong or is there a bug?
https://github.com/XodiumSoftware/XodiumSoftware.github.io

**Comment by oscarotero** [comment, marked as answer]
This is because the `.html` files are copied (with `site.copyRemainingFiles`) so they are not processed by the tailwind plugin.

You can change the extension `index.html` to `index.vto` to load the files (instead of simply being copied) so they will be processed.

FYI, this is a recurrent issue, and I'm thinking of changing this behavior in Lume v3 (more info: https://github.com/lumeland/lume/issues/571) but I didn't find yet a good alternative.

**Reply by illyrius666** [reply]
thank you so much <3
